pom.xml

```diff
@@ -71,8 +71,8 @@
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-compiler-plugin</artifactId>
           <configuration>
-            <source>1.7</source>
-            <target>1.7</target>
+            <source>1.8</source>
```

Comment:
I will downgrade version to 1.7 for the main source, just in case people are embedding it.
But I will kept tests at 1.8 level.

BTW, direct implementation of https://github.com/openzipkin/zipkin-reporter-java can be added in this repo too if not done somewhere else